xbmc/interfaces/python/PythonSwig.cpp.template

```diff
@@ -930,8 +970,17 @@ namespace PythonBindings
     Py_INCREF(&(Ty${it}_Type.pythonType));<%
    }%>
 
-    module = Py_InitModule((char*)"${module.@name}", ${module.@name}_methods);
-    if (module == NULL) return;
+    module = PyModule_Create(&createModule);
+
+    if (module == NULL)
+        return NULL;
+    struct module_state *st = GETSTATE(module);
+
+    st->error = PyErr_NewException((char *)"${module.@name}.Error", NULL, NULL);
```

Comment:
this needs to be removed, I only suggested it for testing but it doesn't do what I thought